subprojects/smoke-test/smoke-test.gradle.kts

```diff
@@ -131,5 +131,8 @@ tasks {
     withType<SmokeTest>().configureEach {
         dependsOn(remoteProjects)
         inputs.property("androidHomeIsSet", System.getenv("ANDROID_HOME") != null)
+        inputs.files(Callable { remoteProjects.map { it.outputDirectory } })
```

Comment:
Are there any other outputs than the output directory? I suppose
```
inputs.files(remoteProjects)
```

would work the same.

Reply:
I would do it explicitly in case we add output directories in the future.